NEWS.md

```diff
@@ -1,5 +1,8 @@
 # roxygen2 5.0.1.9000
 
+* S3 method declarations via setMethodS3() of R.methodS3 and function
+  declarations via setConstructorS3() of R.oo are now supported (#525).
```

Comment:
Can you include your github user name there too?

I don't have a firm policy on issues vs. PRs — but I usually cite the issue number if there is one, other the PR is fine.